README.md

```diff
@@ -8,7 +8,7 @@ using a simpler and newer stack. It's not up-to-par in terms of features, but it
 ### Using the published Docker image 🐳
 
 - `cd` into a training material folder (must have `Slides/slides.json` and `CahierExercices/parts.json`)
-- Run `docker run -it --rm -p 8080:8080 -v $(pwd):/training-material zenika/sensei`.
+- Run `docker run -it --rm -p 8080:8080 -v $(pwd):/training-material zenika/sensei serve`.
```

Comment:
If you add the `serve` command as CMD in the Dockerfile, this is going to be the default behavior when starting the container

```suggestion
- Run `docker run -it --rm -p 8080:8080 -v $(pwd):/training-material zenika/sensei`.
```